### DISCUSSION TITLE
Feasibility of Segmenting Lesions in PET Images Using MONAI Label

### DISCUSSION THREAD

**Discussion by sertacasa** [Project-MONAI/MONAILabel#1603] [General]
Description:
Hello,

I am exploring the capabilities of MONAI Label for a project involving PET images. I would like to know if it's possible to use MONAI Label to segment lesions within these images. If so, could you provide guidance or reference materials that might assist me in setting up and executing this task?

Thank you for your assistance.

**Comment by diazandr3s** [comment]
Hi @sertacasa,

This could be a great use case for MONAI Label.
How many regions do you like to segment? How many segmented samples you've got?
In any case, in this thread you can find all the information to train a MONAI Label model from scratch: https://github.com/Project-MONAI/MONAILabel/discussions/1593#discussioncomment-7754489

Please let us know how that goes,

If you don't mind, I'll move this discussion to the MONAI Label repo.
